lib/sacastats_web/templates/character/lookup.html.heex

```diff
@@ -18,4 +62,16 @@
     </div>
 </div>
 
-<script type="module" src={Routes.static_path(@conn, "/assets/dist/character/lookup.js")}></script>
+<h1 class="text-center mb-5">Favorites</h1>
+<%= if is_nil(@user) do %>
+	<p class="text-center">Login to add favorites!</p>
+<% else %>
+	<%= if @favorite_characters == :error do %>
+        <p class="w-100 mx-auto text-center">An error occured while accessing your favorite characters' statuses from Planetside's Database.</p>
```

Comment:
```suggestion
        <p class="w-100 mx-auto text-center">An error occurred while accessing your favorite characters.</p>
```
small suggestion for the error text

Reply:
Wouldn't it be more correct to mention Planetside's DB as it's not on us that the fetch failed?

Reply:
I don't think it's really relevant for the user to know - they just need to know to try again